DRAFT.md

```diff
@@ -413,6 +413,20 @@ Not marking `operator->` and `operator*` as `noexcept` for `indirect` and
 `polymorphic` would make them strictly less useful than `unique_ptr` in contexts
 where they would otherwise be a valid replacement.
 
+### Explicit constructors
+
+Constructors for `indirect` and `polymorphic` are marked as explicit. This
+disallows “implicit conversion” from single arguments or braced initializers.
+Given both `indirect` and `polymorphic` use dynamically-allocated storage, there
+are no instances where an object could be considered semantically equivalent to
+its constructor arguments (unlike `pair` or `variant`). To construct an
+`indirect` or `polymorphic` object, and with it use dynamically allocate memory,
+the user must explicitly use a constructor.
```

Comment:
Optional: Can mention the standard already does this for the `std::inplace_t` constructors for `std::optional` and `std::any`.

Reply:
Nice idea! Done.